python-sdk/src/astro/constants.py

```diff
@@ -21,6 +21,8 @@ class FileLocation(Enum):
     HTTPS = "https"
     GS = "gs"  # Google Cloud Storage
     S3 = "s3"  # Amazon S3
+    WASB = "wasb"  # Azure Blob Storage
+    WASBS = "wasbs"  # Azure Blob Storage
```

Comment:
dont we need abfss protocol to be added here ?

https://stackoverflow.com/questions/60277545/what-is-the-difference-between-abfss-and-wasbs-in-azure-storage

Reply:
@phanikumv the **WASB** protocol has been marked as **legacy** and may be retired in the upcoming years however, the current [Microsoft Azure Provider](https://airflow.apache.org/docs/apache-airflow-providers-microsoft-azure/stable/index.html) does not support the ADLS2 (ABFSS) protocol.

@mateusoliveiraowshq, @tatiana and I submitted the following [issue](https://github.com/apache/airflow/issues/28223) on the Apache Airflow codebase that implements a new Hook under the Microsoft Azure Provider.

Once we sort that out, the ABFS support is going to be added/implemented on the Astro Python SDK as new issue to be developed.

Reply:
thats good, I think there is a PR already out there for ABFS support https://github.com/apache/airflow/pull/28262